Documentation/coding-guidelines/package-projects.md

```diff
@@ -15,16 +15,16 @@ This is the simplest case.  The package project need only reference the single p
 Sample `System.Text.Encodings.Web.pkgproj`
 ```
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+<Project DefaultTargets="Build">
```

Comment:
Overlapping with your other PR, but you can remove the DefaultTargets and XML prolog here.  Not sure which PR you want to do that in.